From d0914565e7bbd5654ef1d0b5925fafb29269a66b Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Sun, 15 Sep 2024 09:50:31 +0300
Subject: [PATCH] [SW-195710] gaudi3: fix bis pci revision

It was decided that the China SKU version of gaudi3 should be
called Gaudi3D, to resemble the Gaudi2D device that is familiar
to customers.
Respectively the PCI revision of a BIS device should be 4.

Change-Id: I1cc4a745d95cdd605b4fb7b2ed09cb655eabaa0e
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 3cf073a..46535b7 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -979,7 +979,7 @@
 	return !!(regmap_readl(mmD0_PIF_ARC_CONTROL) & PIF_CTRL_BIS_MODE_EN);
 }
 
-#define BIS_DEVICE_PCIE_REVISION	0x2
+#define BIS_DEVICE_PCIE_REVISION	0x4
 
 static void pif_arc_pcie_device_id_vendor_id_set(void)
 {
